### DISCUSSION TITLE
postfix modifications lost on upgrade

### DISCUSSION THREAD

**Discussion by stutteringp0et** [modoboa/modoboa#3196] [Upgrade]
My catchall addresses stopped working after the last upgrade.  I didn't notice for a while, but when I investigated, I found that the one-liner added to /etc/postfix/main.cf was gone.  When adding it back, catchalls work again.

My question is, how can I retain these modifications on upgrade?  Is it something I need to review after every upgrade?

**Comment by tonioo** [comment]
@stutteringp0et Yes. The upgrade process proposed by the installer does not deal with custom modifications to config files

**Reply by stutteringp0et** [reply]
Not that every possible customization should be handled by the installer - but this particular customization, the catchall, which has been discussed many times in issues - couldn't that be a configurable option, or a customization that carries over an upgrade?
